gn3/settings.py

```diff
@@ -24,3 +24,6 @@
 
 # biweight script
 BIWEIGHT_RSCRIPT = "~/genenetwork3/scripts/calculate_biweight.R"
+
+# wgcna script
+WGCNA_RSCRIPT = "wgcna_analysis.R"
```

Comment:
@pjotrp and @zsloan beware of this.

**suggestion:** Best to have it as part of GUIX; and use a relative path like: `GN_GUIX_PATH/bin/wgcna_analysis.R`

Reply:
I will try to hack on that. This is one of the issues that have actually faced relatively and absolute paths.Currently
have had to use hardcoded ones to run the script.